website/docs/events/m3.md

```diff
@@ -0,0 +1,48 @@
+# Workshop: Multiplier Event 3
+
+## Schedule
+
+- Date: Thursday, March 9, 2023
+- Time: 10:00-12:00 EEST
+- Place: Online, on the ["Meeting room" voice channel from the Open Education Hub Discord server](https://discord.gg/bbqhDj3K)
+
+### What We Did
+
+> 10:00 - 10:15
+
+We will deliver a short presentation on the status of Operating Systems course.
+We will describe our methodology, the way we store content, how it's presented, how we integrate quizzes and others.
+
+### Opinions from Stakeholders
+
+> 10:30 - 10:45
+
+Participants will be encouraged to voice their opinions on the Operating Systems course based on the previous presentation.
+The stakeholders range from professors to teaching assistants, industry representatives and other content creators.
```

Comment:
This is where we present the feedback that we have gathered. We do not give the mic to the crowd.